src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -81,15 +132,29 @@ module.exports = function relatedNodes(module) {
  
       if (module.options.suggestRelated) {
         relatedEl.fadeIn();
-        fetchRelated(show);
+        addedRelatedEl.fadeIn();
+        relatedResultShow = true;
+        fetchRelated(showRelatedResult);
       }
  
     });
+
+    $(module.el).find('input').click(function(e) {
+      var showResults = false;
+      showResults = module.options.suggestRelated && relatedResultShow;
```

Comment:
```javascript
   var showResults = false;
   showResults = module.options.suggestRelated && relatedResultShow;
```

Why not declaration and initialization on the same line? It doesn't make sense to initialize this to `false` and then override it?

Reply:
Done